spec/system/admin/enterprises_spec.rb

```diff
@@ -185,7 +186,8 @@
     fill_in 'enterprise_address_attributes_latitude', with: '-37.4713077'
     fill_in 'enterprise_address_attributes_longitude', with: '144.7851531'
     # default country (Australia in this test) should be selected by default
-    select2_select 'Victoria', from: 'enterprise_address_attributes_state_id'
+    page.find("#enterprise_address_attributes_state_id-ts-control").click
+    page.find(".option", text: "Victoria").click
```

Comment:
Why not this way?
```suggestion
    select "Victoria", from: 'enterprise_address_attributes_state_id'
```

Reply:
Well, that works too. But doing the same thing in the other places 
```
        page.find("#enterprise_address_attributes_country_id-ts-control").click
        page.find(".option", text: "Australia").click

        page.find("#enterprise_address_attributes_state_id-ts-control").click
        page.find(".option", text: "Victoria").click
```

is breaking the test for some reason. To just keep things consistent, I think it's better to leave it as is (?)

Reply:
That's a shame. If we keep it as is then it would be good to create a helper for this, in a future pull request. We don't need to change it now. But if a standard `select` doesn't work, it may also point to a usability problem. While most people *click*, maybe the state selection update doesn't work when selecting the country with the keyboard or via voice? It may be worth investing some time to figure out if we can fix it and then we all future devs will save some work figuring this out.